### PR TITLE
event listeners for artist, album, playlist, track buttons, new html

### PR DESCRIPTION
**Pull request by Conecked**

### CODE REVIEW

**Review by Mekken — CHANGES_REQUESTED** [review]
You have to pull the changes from Master and then commit your changes. There are still old method names and variables that are missing in this commit.

File: assets/javascript/main.js [hunk 1]
```diff
@@ -21,19 +24,44 @@ $(document).ready(function () {
             .then(response => response.json())
             .then(resJSON => console.log(resJSON))
 
-            var results = response.data;
+        var results = response.data;
 
-            for (let i = 0; i < results.length; i++) {
-                const element = array[i];
+        for (let i = 0; i < results.length; i++) {
+            const element = array[i];
```
Comment (Mekken) [inline assets/javascript/main.js]: Not sure what this for loop is doing, it's using the results and then putting it into a non existent variable named 'array'. This code isn't used, should be removed.

File: assets/javascript/main.js [hunk 2]
```diff
@@ -21,19 +24,44 @@ $(document).ready(function () {
...
+        event.preventDefault();
+        // authenticate();
+    })
+
+    // TODO make playlist call here
+    $("#playlist").on("click", function (event) {
+        event.preventDefault();
+        // authenticate();
+    })
+
```
Comment (Mekken) [inline assets/javascript/main.js:84]: All of this looks good.

**Comment by Mekken** [comment]
Please look over the comments placed in the "Files Changed" section

File: assets/javascript/main.js (hunk 1 above)
Comment (Conecked) [reply assets/javascript/main.js]: I removed it.

**Review by Mekken — APPROVED** [review]

**Pull request merged**